### PR TITLE
Bugfix/complaint form bugfix

### PR DESCRIPTION
**Pull request by rakeshkumar-verma**
Complaint form bugfix

### CODE REVIEW

File: ui.content/src/main/content/META-INF/vault/filter.xml [hunk 1]
```diff
@@ -10,4 +10,5 @@
 	<filter root="/var/workflow/models/hclecomm" mode="merge"/>
 	<filter root="/content/forms/af/hclecomm" mode="merge"/>
 	<filter root="/content/dam/formsanddocuments/hclecomm" mode="merge"/>
+	<filter root="/content/dam/formsanddocuments-themes/hclecomm" mode="merge"/>
```
Comment (rishabh-behl) [inline ui.content/src/main/content/META-INF/vault/filter.xml:13]: @rakeshkumar-verma  : can we not keep a single locations for forms dam and have folders inside rather at same level like : /content/dam/formsanddocuments/ , /content/dam/formsanddocuments/hclecomm/themes

File: ui.apps/src/main/content/jcr_root/etc/clientlibs/fd/themes/hclecomm/hclecomm-theme/css/theme.css [hunk 2]
```diff
@@ -0,0 +1,1881 @@
+.guideImage img{
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/etc/clientlibs/fd/themes/hclecomm/hclecomm-theme/css/theme.css:1]: @rakeshkumar-verma : what is the source of the css code ?

**Review by rishabh-behl — COMMENTED** [review]
@rakeshkumar-verma  : pl check on comments

File: ui.content/src/main/content/META-INF/vault/filter.xml (hunk 1 above)
Comment (rakeshkumar-verma) [reply ui.content/src/main/content/META-INF/vault/filter.xml:13]: @rishabh-behl This is existing folder structure for af theme and forms.

File: ui.apps/src/main/content/jcr_root/etc/clientlibs/fd/themes/hclecomm/hclecomm-theme/css/theme.css (hunk 2 above)
Comment (rakeshkumar-verma) [reply ui.apps/src/main/content/jcr_root/etc/clientlibs/fd/themes/hclecomm/hclecomm-theme/css/theme.css:1]: @rishabh-behl  this is default generated css, few things i have updated like font- family.

**Pull request merged**